PeerioTechnologies/peerio-mobile
# Feat loading screen animation

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/16061/mobile-loading-animation-update

#### Testing instructions  
There are two screen recordings that can be used for reference, both are Android:
- Emulator (Samsung Galaxy S7)
- Device (Samsung Galaxy S7 Edge)

What to test for:
1- The loading screen needs to match the recordings and the animation example made by design (in CH). This animation only occurs when a user starts the app and is already logged in. Loading screen does not appear on sign up or when manually logging in.
2- Ensure that sign up process still works correctly, specifically what happens when a user finished signing up. They should be transitioned to the welcome screen, then presented with beacons when they click on any tab
3- Ensure that manual login process still works correctly, specifically the transition when a user enters the correct credentials. They should be transitioned to the chats screen (zero state or normal chat list, depending on the whether or not they have chats). Both cases (zero state and non zero state) need to be tested.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
